src/common/error/internalError.ts

```diff
@@ -6,40 +6,41 @@ export enum InternalErrorLevel {
     Warning,
 }
 
+export interface IInternalErrorArgument {
+    argument: any;
+    isPii: boolean;
```

Comment:
I would make isPii optional and consider it to false if it's not specified implicitly.